raiden-ts/src/raiden.ts

```diff
@@ -259,6 +263,26 @@ export class Raiden {
       return { totalSupply, decimals, name, symbol };
     });
 
+    this.getUDCBalance = async () =>
+      from(this.deps.userDepositContract.functions.balances(address))
+        .pipe(
+          map(balance => {
+            const owedAmount = Object.values(state.path.iou)
+              .reduce(
+                (acc, value) =>
+                  acc.concat(
```

Comment:
**_Might_** make sense performance-wise here to use
```
Array.prototype.push.apply(arr1, arr2)
```
instead of `concat()` because `push()` is a lot faster. `arr1.push(...arr2)` seems like a better, shorter ES6-notation for this. I don't know to what dimensions `state.path.iou` can grow but [here's](https://dev.to/uilicious/javascript-array-push-is-945x-faster-than-array-concat-1oki) an article with benchmarks and explanations. Since we want to use the SDK on smaller IoT devices as well it might make sense to consider this